Dockerfile

```diff
@@ -9,6 +9,7 @@ ADD nakadi /nakadi
 RUN find ./nakadi | grep -E "(__pycache__|\.pyc|\.pyo$)" | xargs rm -rf
 
 ADD run.sh /nakadi/run.sh
+ADD scm-source.json /scm-source.json
```

Comment:
I think that it's ok to fail build if scm-source.json is missing. Build script (if it exists) must generate it.